alks_error.go

```diff
@@ -0,0 +1,28 @@
+package alks
+
+import (
+	"fmt"
+)
+
+type AlksError struct {
+	StatusCode int
+	RequestId  string `json:"requestId"`
+	Err        error
+}
+
+func (r *AlksError) Error() string {
```

Comment:
where is this used?

Reply:
Error is an interface, in which you have to define an Error() function.  That function gets called from the terraform provider

Reply:
Is AlksResponseError also meant to be "exportable" or called by the terraform provider?

Reply:
AlksResponseError is just the old AlksError renamed.  There is a defect to refactor error handling for the rest of go-client and tfp, but for now we just wanted to get it started so it can be used in the ltk stories.